jest.config.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
   },
 
   // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
-  // modulePathIgnorePatterns: [],
+  modulePathIgnorePatterns: ["<rootDir>/dist/"],
```

Comment:
because the test is checking the files on `dist` folder

Reply:
yeah. so then my question would be:
a) why are tests ending up in the dist folder? -- let's fix that 
b) if it's just a performance concern, then maybe we should fix it from the other direction and tell it to *only* look in `src`, since if it's scanning dist, it's probably scanning everything else from the root as well.

Reply:
you are right on both questions, I've fixed them.